File: javascript/problem006.js
//Variables
let sum = 0;
let squares = 0;

//Finds sum and sum of squares
for (i = 1; i <= 100; i++) {
    sum += i;
    squares += i**2;
}

//Finds square of sum
sum = sum ** 2;

//Displays answers
console.log(sum - squares); //Expected 25164150

File: java/problem006.java
public class problem006 {
    public static void main(String[] args) {
        //Variables
        int sum = 0;
        int squares = 0;

        //Finds sum and sum of squares
        for (int i = 1; i <= 100; i++) {
            sum += i;
            squares += i * i;
        }

        //Finds square of sum
        sum = sum * sum;

        //Displays answers
        System.out.println(sum - squares); //Expected 25164150
    }
}